sbi/utils/analysis_utils.py

```diff
@@ -84,28 +82,33 @@ def posterior_peaks(samples, return_dict=False, **kwargs):
     else:
         return list(peaks.values())
 
-if __name__ == "__main__":
 
-    '''
-    usage example:
-    '''
-    import torch
-    from sbi import utils as utils
-    from sbi import analysis as analysis
-    from sbi.inference.base import infer
+def get_1d_marginal_peaks_from_kde(
```

Comment:
I'm confused...what's the difference between this and `posterior_peaks` above? Did you maybe forget to delete `posterior_peaks`?